client/my-sites/preview/main.jsx

```diff
@@ -66,23 +70,52 @@ class PreviewMain extends React.Component {
 	}
 
 	render() {
-		const { translate } = this.props;
+		const { translate, isPreviewable, site } = this.props;
+
+		if ( ! site ) {
+			// todo: some loading state?
+			return <span></span>;
```

Comment:
Do you intend to add something to this PR for this output for a loading state? 

Can you just return without the `span` if not, or does that break something?